Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -196,6 +196,13 @@ Note that quantifier variable domains (<- <Domain>) are available in both syntax
     IsHidden = true
   };
 
+  public static readonly Option<bool> RawPointers = new("--raw-pointers", () => false,
+    @"
+false - All class instances are reference-counted or garbage collected
+true - All class instances are raw pointers and need to be manually deallocated".TrimStart()) {
```

Comment:
For now, I've left the option hidden because it's not something we support immediately. And there are precedents of not writing documentations for hidden options.
https://github.com/dafny-lang/dafny/pull/5695/files#r1717185862

However, I do have documentation about these and I'm going to update it as well.

Reply:
OKay if you intend to leave it hidden for now, that's just fine.